.github/workflows/tiles-file.yml

```diff
@@ -0,0 +1,64 @@
+name: 'Deploy Tiles'
+on:
+  workflow_dispatch:
+  pull_request:
+      branches: [ "main" ]
+      paths:
+        - ".github/workflows/deploy-tiles.yml"
```

Comment:
This will run deployment in the pull request context, is that intended?

Reply:
Yea, I left this in for now so any changes to the tiles deployment could be tested before we go to main.